clang/runtime/dpct-rt/include/dpct/memory.hpp

```diff
@@ -1603,6 +1603,23 @@ template <typename T>
 using usm_device_allocator = detail::deprecated::usm_allocator<T, sycl::usm::alloc::shared>;
 } // namespace deprecated
 
+enum class attribute_type {
```

Comment:
Can we this enum class into `pointer_attributes`?

Reply:
Moved to pointer_attributes::type